README.md

```diff
@@ -348,6 +348,11 @@ $mysql->query('CREATE TABLE test ...');
 $mysql->quit();
 ```
 
+This method will gracefully close the connection to the MySQL database
+server once all outstanding commands are completed. See also
+[`close()`)(#close) if you want to force-close the connection without
```

Comment:
Nit: I think you used a wrong closing bracket here

Reply:
Thanks for spotting, updated as suggested :shipit: